exasol_integration_test_docker_environment/lib/base/task_state.py

```diff
@@ -2,11 +2,11 @@
 
 
 class TaskState(Enum):
-    NONE = 0,
-    INIT = 1,
-    AFTER_INIT =2,
-    RUN = 3,
-    FINISHED = 4,
-    CLEANUP = 5,
-    CLEANED = 6,
+    NONE = (0,)
+    INIT = (1,)
+    AFTER_INIT = (2,)
+    RUN = (3,)
+    FINISHED = (4,)
+    CLEANUP = (5,)
+    CLEANED = (6,)
```

Comment:
I believe the values are not supposed to be tuples here

Reply:
Will fix it

Reply:
:warning: It is upgraded like this because they have been tuples in the past!
**Example**
```python
from enum import Enum

class A(Enum):

    one = 1,
    two = 2,

class B(Enum):

    one = 1
    tow = 2

print(type(A.one))
print(type(B.one))
print()
print(type(A.one.value))
print(type(B.one.value))
```
**output:**
```
<enum 'A'>
<enum 'B'>

<class 'tuple'>
<class 'int'>
```